src/roswire/common/__init__.py

```diff
@@ -2,11 +2,11 @@
 from .action import ActionFormat
 from .base import Duration, Time
 from .format import FormatDatabase
+from .launch import ROSLaunchManager
```

Comment:
Should we rename to `LaunchManager`?

Reply:
See above. I think this is ok.

Reply:
Otherwise, we'd have to name all the other ROSxxx things to be consistent, e.g., ROSNodeManager becomes NodeManager.